ReadMe.md

```diff
@@ -1,3 +1,116 @@
 # MAGE SDK for JavaScript (ES5)
```

Comment:
Might be better to say

# MAGE SDK (web)

Since its the title of the readme, we should say in a oneliner what this thing here is all about.